### PR TITLE
HW-57146 java sdk for transfer refunds

### PR DESCRIPTION
**Pull request by denismironchukepam**

### CODE REVIEW

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java [hunk 1]
```diff
@@ -761,6 +753,97 @@ public void testCreateTransferStatusTransition() throws Exception {
...
+        assertThat(returnValue.getClientRefundId(), is("1573548663"));
+        assertThat(returnValue.getSourceToken(), is("act-ba4e8fdd-614b-11e5-af23-0faa28ca7c0f"));
+        assertThat(returnValue.getSourceAmount(), is(50.0));
+        assertThat(returnValue.getSourceCurrency(), is("USD"));
+        assertThat(returnValue.getDestinationToken(), is("usr-3deb34a0-ffd1-487d-8860-6d69435cea6c"));
+        assertThat(returnValue.getDestinationAmount(), is(50.0));
+        assertThat(returnValue.getDestinationCurrency(), is("USD"));
+        assertThat(returnValue.getNotes(), is("Merchant Payment return to Wallet Balance"));
+        assertThat(returnValue.getMemo(), is("TransferReturn123456"));
+        assertThat(returnValue.getCreatedOn(), is(dateFormat.parse("2019-11-12T11:51:05 UTC")));
```
Comment (hrandhawa-hw) [inline src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java]: Is it possible to extract this to a common re-usable method.

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java [hunk 2]
```diff
@@ -2892,6 +2892,231 @@ public void testCreateTransferStatusTransition_successful() throws Exception {
...
+        try {
+            client.createTransferRefund("transferToken",null);
+            fail("Expect HyperwalletException");
+        } catch (HyperwalletException e) {
+            assertThat(e.getErrorCode(), is(nullValue()));
+            assertThat(e.getResponse(), is(nullValue()));
+            assertThat(e.getErrorMessage(), is(equalTo("Transfer Refund is required")));
+            assertThat(e.getMessage(), is(equalTo("Transfer Refund is required")));
+            assertThat(e.getHyperwalletErrors(), is(nullValue()));
+            assertThat(e.getRelatedResources(), is(nullValue()));
```
Comment (hrandhawa-hw) [inline src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java]: Same with this, is it possible to extract this to a common re-usable method.

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java (hunk 1 above)
Comment (denismironchukepam) [reply src/test/java/com/hyperwallet/clientsdk/HyperwalletIT.java]: done

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java (hunk 2 above)
Comment (denismironchukepam) [reply src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java]: done

**Pull request closed without merging**